packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -11,31 +11,44 @@ import { Counter } from './Counter';
 export interface TabProps extends HTMLProps<HTMLLIElement> {
   label: string;
   active?: boolean;
+  /** With this it is possible to use the tab as a hyperlink. Use in cases where the tabs update location. */
+  href?: string;
   icon?: IconName;
+  /** May cause unintended behaviour when used with href */
```

Comment:
uhm, not sure this fits into props table comment. Think this should be a part of the component documentation.